Lesson_1/Task1/Task1/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    internal class program
+    {
+        static void Main(string[] args)
+        {
+            double x, y, z, x1, x2, x3, x4, x5, x6, x7; //здесь объявляются переменные
```

Comment:
You could have only three variables and you could do all math operations on one line for every var, also you could join var initialization and its assigning